bin/build-manifest.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-env node */
+"use strict";
+
+var _ = require("lodash");
+var execSync = require("child_process").execSync;
+var fg = require("fast-glob").sync;
+var fs = require("fs");
+
+/**
+ * Clean up output to ensure that special terms, acronyms,
+ * et cetera are properly formatted.
+ *
+ * @param {String} str Input string on which to perform find and replace.
+ * @return {String} The input string with replacements applied.
+ */
+var replaceValuesInString = function (str) {
+    var replacementPatterns = {
+        "a 11 y": "a11y",
```

Comment:
How do these strange split-up strings arise?

Reply:
@amb26: It's a result of using `lodash.lowercase` to convert file and directory names into sentences. Here's an example (check the console): https://codepen.io/greatislander/pen/LYZBqyg